jha/src/components/SampleImageTipBox.vue

```diff
@@ -52,27 +57,46 @@ export default {
     documentType: {
       type: String,
       default: null,
+    },
+    includeGenderDocumentSamples: {
+      type: Boolean,
+      default: false,
     }
   },
   data: () => {
     return {
       isModalOpen: false,
+      showGenderDocs: false,
+      genderDocsTitle: 'Change of Gender Designation and Request for Waiver of Parental Consent form samples',
+      genderDocsSampleImageFileName: 'gender_forms.png'
```

Comment:
Is there a way you could add this to the `SupportDocumentSamples` in "document-types.js"? Ideally the file names should be in one place.